wfc3tools/meta.yaml

```diff
@@ -6,7 +6,7 @@ build:
     number: '0'
 package:
     name: wfc3tools
-    version: 1.3.1
+    version: 1.3.3
 requirements:
     build:
     - d2to1
```

Comment:
`d2to1` can be safely undefined now that you've moved to the astropy packaging framework ;)